runtime/i386.S

```diff
@@ -150,108 +147,59 @@ LBL(105):
         popl    %esi; CFI_ADJUST(-4)
         popl    %edi; CFI_ADJUST(-4)
         popl    %ebp; CFI_ADJUST(-4)
-    /* Return to caller */
+    /* Return to caller. Returns young_ptr in %eax. */
+        movl    G(Caml_state), %eax
```

Comment:
No need to do this: `%ebx` already contains `Caml_state` here.

Reply:
Changed